src/main/java/com/google/ehub/servlets/ProfileDatastore.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**Manages the User Profiles stored in Datastore.**/
+public final class ProfileDatastore {
+  public static final String PROFILE_ITEM_KIND = "profile";
+  public static final String NAME_PROPERTY_KEY = "name";
+  public static final String EMAIL_PROPERTY_KEY = "email";
+  public static final String USERNAME_PROPERTY_KEY = "username";
+  public static final String BIO_PROPERTY_KEY = "bio";
+
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  public void addUserProfileToDatastore(String Name, String Email, String Username, String Bio) {
```

Comment:
The names of the parameters in this function all seem to start with capital letter which is not a convention that we are following in this codebase. You should rename them so they use camelCase instead.

Reply:
I will make them lowercase